scripts/apps/authoring/metadata/metadata.js

```diff
@@ -1199,16 +1199,21 @@ function MetadataService(api, subscribersService, config, vocabularies, $rootSco
                     return true;
                 } else {
                     cv.terms = cv.items;
-                    return qcodes.length === 0 || qcodes.some((qcode) => !!cvService[qcode]);
+                    return qcodes.some((qcode) => !!cvService[qcode]);
                 }
             }));
         },
         getCustomVocabulariesForArticleHeader: function(qcodes, editor, schema) {
             return this.getFilteredCustomVocabularies(qcodes)
                 .then(
-                    (cvs) => cvs.filter((cv) => cv.items && cv.items.length && (editor[cv._id] || schema[cv._id]))
+                    (cvs) => cvs.filter((cv) => cv.terms.length && (editor[cv._id] || schema[cv._id]))
                 );
         },
+        getAllCustomVocabulariesForArticleHeader: function(editor, schema) {
```

Comment:
Could you add a comment on how `getAllCustomVocabulariesForArticleHeader` and `getCustomVocabulariesForArticleHeader` differ and where are they used?

Reply:
What was the problem for ntb, why weren't the content profiles working?

Reply:
that's not actually fixing those content profiles not working, it's only fixing issue with some cvs displayed in content tab

Reply:
What was then the issue with cvs in the content tab?